programs/mango-v4/src/instructions/token_edit.rs

```diff
@@ -277,8 +277,12 @@ pub fn token_edit(
                 bank.reduce_only,
                 reduce_only
             );
+
             // security admin can only make it stricter
-            if reduce_only < bank.reduce_only {
+            // anything that makes it less strict, should require admin
+            if (bank.reduce_only == 1 && reduce_only == 2)
+                || (bank.reduce_only == 2 && reduce_only == 0)
```

Comment:
this would mean the security admin could change to 0 if it's currently at 1.

Suggestion: `if reduce_only == 0 || (reduce_only == 2 && bank.reduce_only == 1)`
(require admin always for disabling; require admin for weakening from 1 to 2)